### ISSUE TITLE
Home directory never stops loading

### ISSUE THREAD

**Issue by jordanpinski** [redsolver/vup#1]
The home directory never stops loading.

![Screen Shot 2022-04-01 at 10 38 47 AM](https://user-images.githubusercontent.com/8599367/161296430-b08682a2-a243-4f49-9a12-8f4a56d3369a.png)

**Comment by redsolver** [comment]
Does it work after a restart? If it doesn't check the log file (location is shown at the bottom of the settings page)

**Comment by jordanpinski** [comment]
No, restarting doesn't resolve the issue. Here's the log: https://share.vup.app/#skyfs://r:c5UiWlYCTU2-WhHZTC1H892RI4t_eeDgTyTGRs1C_o5bpOJjoiuDHtM_PI4M54uGIWoAQwXxHlIb-F_MDJiQ4w==@7105c9270fa21ec44143555cd98fd09486309dfcd84d54f1b19aff66a3f915e0

On a side note, it's happening every device I've used so far (MacOS & iOS)

**Comment by redsolver** [comment]
hmm the logs don't show any errors and I can't reproduce the issue with a new account on macOS. What exactly does the "Portal Auth" page in the Settings show?

**Comment by jordanpinski** [comment]
Yea that's strange, I think it may have something to do with how old the MySky account is. I generated a new seed & everything works fine. Anyway here's what's showing on the "Portal Auth" settings with the old MySky seed:
![image](https://user-images.githubusercontent.com/8599367/162620154-8f844c9f-3d4a-4909-9887-2494be7801a7.png)

**Comment by redsolver** [comment]
This is likely caused by a recent portal-side change where some skylinks time out. Vup in this case fetches the (valid) registry entry for your home directory, but then times out fetching the skylink stored inside of it. Skynet Labs is now working on solving that, unfortunately this can't be solved client-side.

**Comment by redsolver** [comment]
I encountered the same issue when trying to load some specific directories in my music collection
